src/components/Home/Home.js

```diff
@@ -21,7 +21,7 @@ const actions = [
   {
     id: 'assign-tokens',
     label: 'Assign Tokens',
-    appName: 'Token Manager',
+    appName: 'Tokens',
```

Comment:
Ahh, we should change `appName` to use `appId` instead, and use [these `appId`s](https://github.com/aragon/aragon/blob/master/src/environment.js#L16)